src/server/project.ts

```diff
@@ -500,12 +501,13 @@ namespace ts.server {
             this.projectService.onUpdateLanguageServiceStateForProject(this, /*languageServiceEnabled*/ true);

```

Comment:
You need to set exceededFileName to undefined here.

Reply:
Added. Good catch.